docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,11 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - Logging actions can help to identify bad behaviours, intentional or unintentional, to be spotted by reviewers.
+    Bad behaviours in this context are anything that could compromise the security of the SRE, or result in the accidental disclosure of data outside the TRE.
+    Any measures that TRE developers deem valuable for reporting should be logged.
```

Comment:
@manics @JimMadge now with just the one capability component in this PR, do you think this is still worth including? It's intentionally vague in my opinion, so as to satisfy #59 without being too prescriptive. One option could be to downgrade to Optional

Reply:
There's not problem with including this with one component, but I don't think this wording quite addresses @crickpetebarnsley's point:

> How can a research environment be trusted if the logs of who did what when and with what data are not publicly available? Or if the network connection attempts are rising or people are failing to login due successfully. Transparency must be a principle for a environment that is trusted for research.

Instead of saying TREs should log "user actions" how about saying that actions involving access to the data should be logged? Maybe think about what's worth logging (eg. user logins, data/code being ingressed/egressed, etc.) and exclude things that aren't worth logging (the exact keystrokes that users type at a terminal). Think about what you would want to see to be convinced that someone else had run a robust analysis on some data.